src/node/README.md

```diff
@@ -1,7 +1,7 @@
 
-# Node.js (via nvm) and yarn (node)
+# Node.js (via nvm), yarn and pnpm (node)
 
-Installs Node.js, nvm, yarn, and needed dependencies.
+Installs Node.js, nvm, yarn, pnpm, and needed dependencies.
```

Comment:
Same here. Can you update 👇 instead? 

https://github.com/devcontainers/features/blob/c8590bef3dce55bfc110fd3abdf6cd28e7d7e93f/src/node/devcontainer-feature.json#L6